### ISSUE TITLE
ProcessConfig.initialise() under netcore throws Exception

### ISSUE THREAD

**Issue by stefc** [louthy/language-ext#157]
I want to simply test the Actormodel under netcore but after a call of 
 ProcessConfig.initialise();

I get the following exception call stack : 
It seems that some key in a dictionary (providers?) not exists ?

```
Unhandled Exception: System.Exception: Key doesn't exist in map
   at LanguageExt.Prelude.failwith[R](String message)
   at LanguageExt.Option`1.Match[R](Func`2 Some, Func`1 None)
   at LanguageExt.Map`2.get_Item(K key)
   at LanguageExt.ActorSystem..ctor(SystemName systemName, Option`1 cluster, AppProfile appProfile, ProcessSystemConfig settings)
   at LanguageExt.ActorContext.StartSystem(SystemName system, Option`1 cluster, AppProfile appProfile, ProcessSystemConfig config)
   at LanguageExt.ProcessConfig.<>c__DisplayClass4_0.<StartFromConfig>b__8()
   at LanguageExt.Option`1.Match(Action`1 Some, Action None)
   at LanguageExt.ProcessConfig.StartFromConfig(ProcessSystemConfig config)
   at MapExtensions.Iter[K,V](Map`2 self, Action`1 action)
   at LanguageExt.Option`1.Match[R](Func`2 Some, Func`1 None)
   at LanguageExt.ProcessConfig.initialise(String configText, Option`1 nodeName, Action setup, IEnumerable`1 strategyFuncs)
   at LanguageExt.ProcessConfig.initialise()
   at txs.actor.Program.Main(String[] args)

```

**Comment by louthy** [comment]
Hi @stefc, what OS and version are you running this on?

**Comment by stefc** [comment]
Windows 7 with dotnet core 

dotnet --versions give me the following

```
Version 1.0.0-preview2-003121
OS Name : Windows
OS Version: 6.1.7601
RID: win7-x64

```
LanguageExt.Core / .Process package is 1.9.3

**Comment by louthy** [comment]
Ok, looks like a .NET Core specific problem.  I've managed to repro from
the package, I'll try and take a look this evening.

**Comment by louthy** [comment]
@stefc This should be resolved on [v1.9.4 on nu-get](https://www.nuget.org/packages/LanguageExt.Process).

**Comment by stefc** [comment]
Work very good now with netcore ! Thx for the fix.
